### PR TITLE
fix(ext/node): use primordials in `ext/node/polyfills/_brotli.js`

### PR DESCRIPTION
**Pull request by iuioiua**
Towards #24236

### CODE REVIEW

File: ext/node/polyfills/_brotli.js [hunk 1]
```diff
@@ -28,7 +34,9 @@ const toU8 = (input) => {
     return enc.encode(input);
   }
 
+  // deno-lint-ignore prefer-primordials
   if (input.buffer) {
+    // deno-lint-ignore prefer-primordials
     return new Uint8Array(input.buffer);
   }
```
Comment (iuioiua) [inline ext/node/polyfills/_brotli.js:41]: Note: using `TypedArrayPrototypeGetBuffer` here causes `node_unit_tests::zlib_test` to fail. Hence the ignores.

Comment (iuioiua) [reply ext/node/polyfills/_brotli.js:41]: @petamoriken, does this look right to you?

Comment (petamoriken) [reply ext/node/polyfills/_brotli.js:41]: Maybe it is necessary to detect TypedArray or DataView.

https://github.com/denoland/deno/blob/8dd7beb7c6b775d0952c6be0e458066d4cb7ad09/ext/web/08_text_encoding.js#L125-L131

Comment (iuioiua) [reply ext/node/polyfills/_brotli.js]: That makes sense and worked! Thank you.

**Review by satyarohith — APPROVED** [review]
LGTM

**Pull request merged**